resources/qml/Preferences/GeneralPage.qml

```diff
@@ -377,6 +379,40 @@ UM.PreferencesPage
                 }
             }
 
+            Item
+            {
+                //: Spacer
+                height: UM.Theme.getSize("default_margin").height
+                width: UM.Theme.getSize("default_margin").width
+            }
+
+            Label
+            {
+                font.bold: true
+                text: catalog.i18nc("@label", "Override Profile")
+            }
+
+            UM.TooltipArea
+            {
+                width: childrenRect.width;
+                height: childrenRect.height;
+
+                text: catalog.i18nc("@info:tooltip", ".")
+
+                ComboBox
+                {
+                    id: choiceOnProfileOverrideDropDownButton
+
+                    model: [
+                        "Always ask me this",
```

Comment:
These strings are now never translated. You should use the catalog so that they can be internationalised.